src/routes/polls.js

```diff
@@ -42,6 +42,15 @@ module.exports = function(app) {
     })
   );
 
+  app.get(
```

Comment:
Since this is modifying a poll, thoughts on using a `PUT` instead of `GET` ?  `app.put` I believe.